### PR TITLE
+ Presto, Java version update, plus a ton of smaller changes

### PR DESCRIPTION
**Pull request by akshaylive**
Updating presto version to 0.128. This version of presto requires Java 8. Modified dependencies. Testing image at: "docker pull akshaylive/presto-marathon"

### CODE REVIEW

**Comment by akshaylive** [comment]
@sheepkiller could you review/close this please?

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 ## Howto
 ### Setup
 ```sh
-docker pull sheepkiller/presto-marathon
+docker pull akshaylive/presto-marathon
```
Comment (sheepkiller) [inline README.md]: Could you please revert this change ? Automatic build is configured

File: docker/presto/Dockerfile [hunk 2]
```diff
@@ -1,14 +1,16 @@
-FROM centos:centos6
+FROM centos:centos7
 
 MAINTAINER Clement Laforet <sheepkiller@cultdeadshee.org>
+MAINTAINER Akshaya Shanbhogue <akshay.live@gmail.com>
```
Comment (sheepkiller) [inline docker/presto/Dockerfile]: Could you please remove MAINTAINER line ? Feel free to add headers to add you.

**Comment by sheepkiller** [comment]
Hi @akshaylive,
Thanks for your patch and sorry for the late reply/review. I've commented your patch(es), if you still want to update your PR.

File: README.md (hunk 1 above)
Comment (akshaylive) [inline README.md]: Oh yes, I added it as there was no activity on this for a while

**Comment by akshaylive** [comment]
@sheepkiller, sorry for the delay this time. The changes have been made. Feel free to merge.

**Comment by szwed** [comment]
@sheepkiller can you merge it? I would like to avoid unnecessary forking :)
